cognite/client/_cognite_client.py

```diff
@@ -62,15 +62,15 @@ def __init__(self, config: Optional[ClientConfig] = None) -> None:
         self.three_d = ThreeDAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
         self.labels = LabelsAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
         self.relationships = RelationshipsAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
-        self.entity_matching = EntityMatchingAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
+        self.entity_matching = EntityMatchingAPI(self._config, api_version="beta", cognite_client=self)
         self.templates = TemplatesAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
         self.vision = VisionAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
         self.extraction_pipelines = ExtractionPipelinesAPI(self._config, api_version="playground", cognite_client=self)
         self.extraction_pipeline_runs = ExtractionPipelineRunsAPI(
             self._config, api_version="playground", cognite_client=self
         )
         self.transformations = TransformationsAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
-        self.diagrams = DiagramsAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
+        self.diagrams = DiagramsAPI(self._config, api_version="beta", cognite_client=self)
```

Comment:
beta APIs should only be made available through the cognite.client.beta.CogniteClient class.

Reply:
https://github.com/cognitedata/cognite-sdk-python/pull/1029/commits/d6d62412e7e44915e848a30ed487157446347709